dataform/ecom-report-transform/index.js

```diff
@@ -0,0 +1,101 @@
+const { getEventParam } = require('./eventParams.js')
+const {
+  getNumericDateFormat,
+  getHistoricalTableName,
+} = require('@optimics/dataform-tables')
+
+/** Return intraday table specific for utm source customers as a string
+ * @param config.ga4Dataset The Fully Qualified dataset name prefixed with GCP project name
+ * @example 'my-domain.cz'
+ * @param config.utmSource The UTM Source, as it is received from URL
+ * @param config.rangeCap How many days into the past should we download
+ * @example 'my-project.my-dataset'
```

Comment:
You are providing an example for `config.rangeCap` with the value of `'my-project.my-dataset'`. This looks incorrect.